server/index.html

```diff
@@ -3,38 +3,146 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>WebSockets</title>
+
+  <link rel="stylesheet" href="../css/styles.min.css">
```

Comment:
I finally finished listening to my wmr podcast and decided to try out parcel, and see if it can be applied to your project. Turns out it can.

Following https://parceljs.org/getting_started.html, it was a case of:
* `yarn global add parcel-bundler`
* change this line to `../scss/styles.scss`
* `yarn node server/index.js` in another tab
* `parcel server/index.html` (this compiles the scss for you, and also hot-reloads the browser when you save any of the files in your import tree from index.html (so all of the scss files, but not server/index.js))

you could potentially run `parcel build server/index.html` as part of your CI pipeline, and it will explode if you make a syntax error in your scss (you might want to use a dedicated linter for your scss as well though)